pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+addopts = -vvv --cov=sample_fast_api --cov-report=term-missing
```

Comment:
This file already exists under the tests folder. Is it realy necessary to duplicate it here?

Reply:
Sorry, i forgot delete the other

Reply:
in the versions above 6+, we can use it directly on pyproject.toml [1], maybe it can be a better choice to avoid on file more

1 - https://docs.pytest.org/en/stable/customize.html#pyproject-toml

Reply:
Nice idea